src/pages/index.js

```diff
@@ -170,16 +242,187 @@ profileCard.setEventListeners();
+        const card = renderCard(res);
+        sectionRenderer.addItem(card);
+        newCard.close();
+      })
+      .catch((err) => {
+        console.error("an error has occurred", err);
+      })
+      .finally(() => {
+        console.log("done");
+      });
   },
 });
 
 newCard.setEventListeners();
+
+/* -------------------------------------------------------------------------- */
+/*                             confirmation popup                             */
+/* -------------------------------------------------------------------------- */
+
+const popupConfirm = new PopupWithConfirmation({
+  popupSelector: "#confirm-delete-modal",
+});
+
+function handleDeleteButton(card) {
+  popupConfirm.setSubmitAction(() => {
+    api
+      .confirmDeleteButton(card._id)
+      .then(() => {
+        card.handleDeleteCard();
+      })
+      .catch((err) => {
+        console.error("an error has occurred", err);
+      })
+      .finally(() => {
+        console.log("done");
+      });
+  });
+  popupConfirm.open();
+}
+
+popupConfirm.setEventListeners();
+
+/* -------------------------------------------------------------------------- */
+/*                   api call for the section card renderer                   */
+/* -------------------------------------------------------------------------- */
+
+let sectionRenderer;
+
+api
+  .getInitialCards()
+  .then((initialCards) => {
+    sectionRenderer = new Section(
+      {
+        items: initialCards,
+        renderer: (cardData) => {
+          const card = renderCard(cardData);
+          sectionRenderer.addItem(card);
+        },
+      },
+      cardListSelector
+    );
+    sectionRenderer.renderItems();
+    const cards = initialCards;
+  })
+  //need to figure out a way to take the ids and plug them into the url. Do i use an event listener of every card and everytime i click a specific one i get the id for it i do i just listen for every time?
+  .catch((err) => {
+    console.error("An error was found", err);
+  });
+
+/* -------------------------------------------------------------------------- */
+/*                                fetch request                               */
+/* -------------------------------------------------------------------------- */
+api
+  .userProfileInfo()
+  .then((res) => res.json())
+  .then((info) => {
+    console.log(info);
+    userInfo.setUserInfo({
+      name: info.name,
+      job: info.about,
+    });
+    console.log(userInfo);
+  })
+  .catch((err) => {
+    console.error("an error has occurred", err);
+  })
+  .finally(() => {
+    console.log("done");
+  });
+
+/* -------------------------------------------------------------------------- */
+/*                                card requests                               */
+/* -------------------------------------------------------------------------- */
+
+function handleLikeButton(card) {
+  if (card._isLiked) {
```

Comment:
`_isLiked`

Please, delete the **low dash** from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.